geth/api/backend_jail_test.go

```diff
@@ -644,8 +645,10 @@ func (s *BackendTestSuite) TestJailWhisper() {
 	for testKey, filter := range installedFilters {
 		if filter != "" {
 			s.T().Logf("filter found: %v", filter)
-			for _, message := range whisperAPI.GetNewSubscriptionMessages(filter) {
-				s.T().Logf("message found: %s", gethcommon.FromHex(message.Payload))
+			messages, err := whisperAPI.GetFilterMessages(filter)
```

Comment:
This is a change in [Whisper V5 API](https://github.com/ethereum/go-ethereum/wiki/Whisper-v5-RPC-API).